PYME/LMVis/Extras/pointSetGeneration.py

```diff
@@ -139,37 +170,83 @@ def refresh_choices(self):
 
 
 
-
-
 class Generator(HasTraits):
     meanIntensity = Float(500)
     meanDuration = Float(3)
     backgroundIntensity = Float(300)
     meanEventNumber = Float(2)
     scaleFactor = Float(2)
     meanTime= Float(2000)
+    mode = Enum(['STORM','PAINT'])
 
     sources = List([WormlikeSource(), ImageSource(), FileSource()])
 
     source = Instance(PointSource)
-    
+
+    helpInfo = {
+        'source' : '''
+Select the type of point source to generate points.
+A wormlike source, an image based source and a file based source are supported.
+''',
+        'meanIntensity' : '''
+This parameter specifies the mean number of photons in an event.
+Typically values are in the range from 100 to several 10000.
+''',
+        'meanDuration' : '''
+The mean duration of events which is specified in units of frames.
+''',
+        'meanTime' : '''
+This parameter, the mean time of the series in frame units, is related to the series duration.
+For PAINT mode this is half the duration of the series, for STORM simulation mode the relationship is a little more complex.
+There it has something to do with the time where most of the events are occuring. 
+''',
+        'meanEventNumber' : '''
+This parameter specifies the mean number of times an event occurs at a single marker location.
+''',
+        'backgroundIntensity' : '''
+The background intensity in units of photons, typically in the range from a few tens to hundreds of photons.
+''',
+        'mode' : '''
+With the simulation mode you can choose between STORM or PAINT mode. 
+This parameter effects how event rate changes with time (it stays constant in PAINT mode).
+It has little effect otherwise.
```

Comment:
```suggestion

```

It's only effect is to change the distribution from which event times are chosen - STORM=exponential, PAINT=uniform.